torchci/pages/api/drci/drci.ts

```diff
@@ -74,7 +74,7 @@ export default async function handler(
     const failures = await updateDrciComments(
       octokit,
       repo,
-      prNumber as string
+      prNumber ? [prNumber as unknown as number] : []
```

Comment:
What does `as unknown as number` do? Is that different from just `as number`?

Reply:
I'm not allowed to cast directly from whatever prNumber is (I think string?) to number, so I have to go through unknown or any first.  You can see an error pop up if you just do as number